jib-cli/src/integration-test/java/com/google/cloud/tools/jib/cli/cli2/JarCommandTest.java

```diff
@@ -64,11 +68,20 @@ public void testErrorLogging_directoryGiven() throws URISyntaxException {
 
   @Test
   public void testJar_toDocker() throws IOException, InterruptedException, URISyntaxException {
-    Path jarFile = Paths.get(Resources.getResource("simpleJar.jar").toURI());
+    Path jarPath = Paths.get(Resources.getResource("jarTest/jarWithCp.jar").toURI());
     Integer exitCode =
         new CommandLine(new JibCli())
-            .execute("--target", "docker://jib-cli-image", "jar", jarFile.toString());
+            .execute("--target", "docker://jib-cli-image", "jar", jarPath.toString());
     String output = new Command("docker", "run", "--rm", "jib-cli-image").run();
+    String classPath = null;
+    try (JarFile jarFile = new JarFile(jarPath.toFile())) {
+      classPath = jarFile.getManifest().getMainAttributes().getValue(Attributes.Name.CLASS_PATH);
+    }
+
+    // Validate classpath before checking output
+    assertThat(classPath).isNotNull();
+    List<String> dependencies = Splitter.onPattern("\\s+").splitToList(classPath.trim());
```

Comment:
No need for splitting. I think it can be `assertThat(classPath).isEqualTo("dependency1.jar;directory/dependency2.jar")`, even without checking not null.

And just double-check that the main in `simplJar` does use classes in both `dependency1.jar` and `dependency2.jar`.